Objects/memoryobject.c

```diff
@@ -1086,6 +1086,16 @@ PyBuffer_ToContiguous(void *buf, const Py_buffer *src, Py_ssize_t len, char orde
     return ret;
 }
 
+static inline Py_ssize_t
+get_exports(PyMemoryViewObject *buf)
+{
+#ifdef Py_GIL_DISABLED
+    return _Py_atomic_load_ssize_relaxed(&buf->exports);
+#else
+    return buf->exports;
+#endif
+}
```

Comment:
```suggestion
#ifdef Py_GIL_DISABLED
static inline Py_ssize_t
get_exports(PyMemoryViewObject *buf)
{
    return _Py_atomic_load_ssize_relaxed(&buf->exports);
}
#else
#define get_exports(buf) buf->exports
#endif
```

This could save us an inlined function call in default builds (having `inline` does not mean that the compiler will really inline it). But I'm not sure whether it would save us a lot.

Reply:
I'm fine with this, but FWIW, I'm pretty sure that any modern compiler will inline a `static inline` function.

Reply:
For simplicitly, let's keep this as is.